app/models/binary_blob/purging.rb

```diff
@@ -0,0 +1,20 @@
+class BinaryBlob < ApplicationRecord
+  module Purging
+    extend ActiveSupport::Concern
+    include PurgingMixin
+
+    module ClassMethods
+      def purge_timer
+        purge_queue(:scope)
+      end
+
+      def purge_window_size
+        ::Settings.binary_blob.purge_window_size
+      end
+
+      def purge_scope(_older_than = nil)
+        where(:resource => nil)
```

Comment:
from what I saw you needed to define something ```purge_related``` since we need to also clean up BinaryBlobPart table, that BinaryBlob uses for storing data. Since purging calls delete_all, the hooks will not be invoked, to clear them, right?

Reply:
Yes, @Ladas is right here.